config/database.yml

```diff
@@ -27,8 +27,8 @@ development:
   <<: *default
   database: hunterskeepers_development
   host: <%= ENV['DB_HOST'] || 'localhost' %>
-  username: <%= ENV['POSTGRES_USER'] || 'root' %>
-  password: <%= ENV['POSTGRES_PASSWORD'] || nil %>
+  username: <%= ENV['POSTGRES_USER'] || "itox" %>
```

Comment:
Why did you rename these??

Reply:
Sorry, I didn't notice I pushed my Postgres config.